tasks/app/controllers/tasks_controller.rb

```diff
@@ -25,6 +25,29 @@ def destroy
     redirect_to "/"
   end
 
+  def edit
+    id = params[:id]
+    @task = Task.find(id)
+  end
+
+  def update
+    id = params[:id]
+    task = Task.find(id)
+    task.update(name: task_params[:task][:name], description: task_params[:task][:description])
```

Comment:
Because we have the `task_params` method, we can simplify this line to `task.update(task_params[:task])`.